src/library_wasmfs.js

```diff
@@ -3,44 +3,50 @@ var WasmfsLibrary = {
   $wasmFS$JSMemoryFreeList: [],
   $wasmFS$preloadedFiles: [],
   $wasmFS$preloadedDirs: [],
-  $FS__deps: ['$wasmFS$preloadedFiles', '$wasmFS$preloadedDirs', '$wasmFS$JSMemoryFiles', '$wasmFS$JSMemoryFreeList'],
+  $FS__deps: [
+    '$wasmFS$preloadedFiles',
+    '$wasmFS$preloadedDirs',
+    '$wasmFS$JSMemoryFiles',
+    '$wasmFS$JSMemoryFreeList',
+    '$asyncLoad',
+  ],
   $FS : {
     // TODO: Clean up the following functions - currently copied from library_fs.js directly.
     createPreloadedFile: function(parent, name, url, canRead, canWrite, onload, onerror, dontCreateFile, canOwn, preFinish) {
-        Browser.init(); 
```

Comment:
Do you know why `library_fs.js` calls this?   Should we remove it from there too for parity?

Reply:
There is a TODO on it there to remove it, but I think it would require some work to actually do. There is some interaction between FS and Browser and their ordering, basically. I think in wasmfs we will not have any of that complexity...